ipv8/REST/attestation_endpoint.py

```diff
@@ -55,6 +58,18 @@ def on_attestation_complete(self, for_peer, attribute_name, attribute_hash, from
             self.identity_overlay.add_known_hash(attribute_hash, attribute_name, for_peer.public_key.key_to_bin(),
                                                  metadata)
 
+    @inlineCallbacks
```

Comment:
Making this method an `inlineCallback` seems a bit unnecessary to me. I would rather just return the deferred.

Reply:
This is consistent with the rest of the code. We can make enhancements like that later in a different PR.

Reply:
Ok then!